extensions/wikia/LookupContribs/SpecialLookupContribs_helper.php

```diff
@@ -259,129 +232,51 @@ private function addWikiaInfo( $wikiaIds, &$userActivity ) {
 		}
 	}
 
-	private function getActivityCount( DatabaseBase $dbr, Array $where ) {
-		$res = $dbr->select(
-			'events',
-			[ 'count(distinct wiki_id) as num' ],
-			$where,
-			__METHOD__
-		);
-
-		$row = $dbr->fetchObject( $res );
-		return $row->num;
+	private function getUserActivityMemKey() {
+		return wfSharedMemcKey( $this->mUserId, 'data' );
 	}
 
-	private function getUserActivityMemKey( $addEditCount ) {
-		return wfSharedMemcKey( $this->mUserId, ( $addEditCount ? 'dataWithEdits' : 'data' ) );
-	}
+	private function orderData( $userActivity ) {
+		$data = $userActivity['data'];
 
-	private function orderData( &$data, $order, $edits ) {
-		$aTemp = [];
-		$aMatches = [];
-		if ( !$order ) {
-			$order = ( $edits ? 'edits:desc' : 'lastedit:desc' );
+		if ( empty( $data ) || !is_array( $data ) ) {
+			return $userActivity;
 		}
-		if ( isset( $data['data'] ) && is_array( $data['data'] ) ) {
 
-			// order by title
-			if ( preg_match( '/^title:(asc|desc)$/', $order, $aMatches ) ) {
-				foreach ( $data['data'] as $aItem ) {
-					$aTemp[$aItem['title']] = $aItem;
-				}
-			}
+		usort( $data, [ $this, 'sortUserActivity' ] );
 
-			// order by url
-			elseif ( preg_match( '/^url:(asc|desc)$/', $order, $aMatches ) ) {
-				foreach ( $data['data'] as $aItem ) {
-					$aTemp[$aItem['url']] = $aItem;
-				}
-			}
-
-			// order by last edit
-			elseif ( preg_match( '/^lastedit:(asc|desc)$/', $order, $aMatches ) ) {
-				foreach ( $data['data'] as $aItem ) {
-					// added URL part since last edits are ints and might not be unique
```

Comment:
Are we sure we want to remove URL from sorting key?
I completely have no idea if it was done for purpose...